sink/datadog.go

```diff
@@ -292,9 +281,10 @@ func (s *Datadog) Send(ctx context.Context, m *blip.Metrics) error {
 			case blip.CUMULATIVE_COUNTER, blip.DELTA_COUNTER:
 				metricVal := metrics[i].Value
 
-				if s.sendCounterDelta && metrics[i].Type == blip.CUMULATIVE_COUNTER {
+				if metrics[i].Type == blip.CUMULATIVE_COUNTER {
 					// if the value is not specifically marked as DELTA, calculate delta
-					metricID := s.uniqIdentifierForMetric(name, tags)
+
+					metricID := s.uniqIdentifierForMetric(name, metrics[i].Group)
```

Comment:
If we are going to use just `Group` and not `Meta` as well then we need to review some of the existing metrics as some of them use `Meta` instead of `Group` for tags that uniquely identify the metric values.

Reply:
Discussed offline. We are going to exclude `meta` for now. The metrics which currently uses meta to uniquely identify values are `percona.qrt` and `query.response-time` which are both guages.

Dan mentioned: by design meta isn't supposed to identify, identity comes from domain + metric + group (if any).